app/src/main/java/id/rizmaulana/covid19/ui/dailygraph/DailyGraphFragment.kt

```diff
@@ -0,0 +1,175 @@
+package id.rizmaulana.covid19.ui.dailygraph
+
+import android.content.Context
+import android.content.res.Configuration
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.annotation.ColorRes
+import com.github.mikephil.charting.components.XAxis
+import com.github.mikephil.charting.data.Entry
+import com.github.mikephil.charting.data.LineData
+import com.github.mikephil.charting.data.LineDataSet
+import com.github.mikephil.charting.formatter.IndexAxisValueFormatter
+import com.google.android.material.tabs.TabLayout
+import id.rizmaulana.covid19.R
+import id.rizmaulana.covid19.data.model.CovidDaily
+import id.rizmaulana.covid19.databinding.FragmentDailyGraphBinding
+import id.rizmaulana.covid19.ui.base.BaseFragment
+import id.rizmaulana.covid19.util.NumberUtils
+import id.rizmaulana.covid19.util.ext.color
+import id.rizmaulana.covid19.util.ext.observe
+import org.koin.android.viewmodel.ext.android.sharedViewModel
+import java.lang.ClassCastException
+
+const val TOTAL_STATE = 0
+const val DELTA_STATE = 1
+
+class DailyGraphFragment : BaseFragment() {
+
+    private val viewModel by sharedViewModel<DailyGraphViewModel>()
+
+    private var _binding: FragmentDailyGraphBinding? = null
+    private val binding get() = _binding!!
+
+    private var currentState = TOTAL_STATE
+    var listener: DailyListener? = null
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        _binding = FragmentDailyGraphBinding.inflate(inflater, container, false)
+        return binding.root
```

Comment:
This is too, same as comment in `DailyDataListFragment`

Reply:
see prev comments